src/Core/Parser/TemplateParser.php

```diff
@@ -531,7 +531,7 @@ protected function isArgumentEscaped(ViewHelperInterface $viewHelper, ?ArgumentD
         $isBoolean = $hasDefinition && ($argumentDefinition->getType() === 'boolean' || $argumentDefinition->getType() === 'bool');
         $escapingEnabled = $this->configuration->isViewHelperArgumentEscapingEnabled();
         $isArgumentEscaped = $hasDefinition && $argumentDefinition->getEscape() === true;
-        $isContentArgument = $hasDefinition && method_exists($viewHelper, 'resolveContentArgumentName') && $argumentDefinition->getName() === $viewHelper->resolveContentArgumentName();
+        $isContentArgument = $hasDefinition && $argumentDefinition->getName() === $viewHelper->getContentArgumentName();
```

Comment:
maybe keep the method_exists() since it's not in the interface? this should happify phpstan ...

Reply:
My thinking was: Before it wasn't in `AbstractViewHelper`, only in the trait, so it made sense to check for its existence. Now it's in the abstract class every ViewHelper extends from, just like `$viewHelper->isChildrenEscapingEnabled()` two lines below it, which also has an entry in phpstan baseline. So at least it's consistent.

Reply:
fine with me. we'll have to resolve these at some point anyways. always two they are ;)